101-cloud-shell-vnet/README.md

```diff
@@ -0,0 +1,31 @@
+# Sample name
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-cloud-shell-vnet/PublicLastTestDate.svg)
+![Azure Public Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/101-cloud-shell-vnet/PublicDeployment.svg)
+
+![Azure US Gov Last Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-cloud-shell-vnet/FairfaxLastTestDate.svg)
+![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/101-cloud-shell-vnet/FairfaxDeployment.svg)
+
+![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/101-cloud-shell-vnet/BestPracticeResult.svg)
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/101-cloud-shell-vnet/CredScanResult.svg)
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cloud-shell-vnet%2Fazuredeploy.json)
+[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cloud-shell-vnet%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cloud-shell-vnet%2Fazuredeploy.json)
+
+This template deploys the necessary resources to run Azure Cloud Shell inside a customer owned virtual network. Azure Cloud Shell  is a browser-based, authenticated, command-line experience based in the Azure Portal for managing cloud resources. Running Cloud Shell in a customer virtual network allows the customer to manage resources that may be isolated from the public internet in a private virtual network.
+
+## Prerequisites
+
+In order to use this deployment there must be an existing resource group and a virtual network. Most users will already have a deesired resource group and virtual network they would like to connect to. If these resources do not exist, they must be created prior to running this template. Both the resource group and the virtual network must be in the same location.
+Important! While this functionality is in preview, only the following locations may be used: WestCentralUS, WestUS 
```

Comment:
what about Azure Gov?  (assuming it's supposed to work there?)

Also use allowedValues on the location if we're restricting locations

Reply:
Azure Gov is coming soon.  Not available yet